src/python/pants/backend/docker/subsystems/dockerfile_parser.py

```diff
@@ -114,11 +115,17 @@ async def setup_process_for_parse_dockerfile(
 
 @dataclass(frozen=True)
 class DockerfileInfo:
+    digest: Digest
     source: str = ""
```

Comment:
Probably shouldn't have a default value, as that would be an error to use "" in production. I think clarity is more important than slightly less boilerplate in tests